src/test/java/uk/gov/justice/probation/courtcaseservice/service/CourtCaseServiceTest.java

```diff
@@ -74,18 +67,20 @@
     @Mock
     private CourtCaseRepository courtCaseRepository;
     @Mock
+    private GroupedOffenderMatchRepository groupedOffenderMatchRepository;
+    @Mock
     private CourtEntity courtEntity;
     @Mock
     private List<CourtCaseEntity> caseList;
     @Mock
     private TelemetryService telemetryService;
     @Captor
-    private ArgumentCaptor<CourtCaseEntity> courtCaseCaptor;
+    ArgumentCaptor<GroupedOffenderMatchesEntity> matchesCaptor;
 
     private CourtCaseEntity courtCase;
 
     @InjectMocks
-    private CourtCaseService service;
+    private ImmutableCourtCaseService service;
```

Comment:
We should either rename this class to ImmutableCourtCaseServiceTest or the underlying service implementation, removing the "Immutable".